### PR TITLE
Debrand the client

### PR DESCRIPTION
**Pull request by Sheikah45**

### CODE REVIEW

File: src/icomoon/Downlord's FAF Client.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "metadata": {
-    "name": "Downlord's FAF Client",
+    "name": "FAF Client",
```
Comment (micheljung) [inline src/icomoon/Downlord's FAF Client.json:3]: File can be renamed as well

**Comment by micheljung** [comment]
There are additional places like installation path etc. which I also changed in the update branch (see #1753)

**Comment by Sheikah45** [comment]
@micheljung how did you create the splash screen initially that needs to be changed as well. I see the png and psd files

**Comment by Sheikah45** [comment]
Alright all the places that can be changed should be changed now

**Comment by Sheikah45** [comment]
Note the user agent cannot be changed as the website will then include the header and footer

**Comment by micheljung** [comment]
> @micheljung how did you create the splash screen initially that needs to be changed as well. I see the png and psd files

Open the PSD in Photoshop, edit and save the PNG :) let me know if you don't have it

**Comment by Sheikah45** [comment]
Yep already done. Just had to look around for a font match

**Pull request merged**